backend/models/user.js

```diff
@@ -24,6 +25,17 @@ const userSchema = new Schema(
 
 // static signup method
 userSchema.statics.signup = async function (email, username, password) {
+  // Validation
+  if (!email || !username || !password) {
+    throw Error("All fields must be filed");
+  }
+  if (!validator.isEmail(email)) {
+    throw Error("The email provided is not valid");
+  }
+  if (!validator.isStrongPassword(password)) {
+    throw Error("The password provided is not strong enough");
```

Comment:
From the [validator]() documentation, the defaults are
```
{ minLength: 8, minLowercase: 1, minUppercase: 1, minNumbers: 1, minSymbols: 1, returnScore: false, pointsPerUnique: 1, pointsPerRepeat: 0.5, pointsForContainingLower: 10, pointsForContainingUpper: 10, pointsForContainingNumber: 10, pointsForContainingSymbol: 10 }
```
Of course, it can be changed if needed.

Reply:
Maybe it would be a good thing to write these requirements under the password input field on the frontend, to let the user know what characters he has to use to create its password! *(or is it already done by the framework?)*

Reply:
I added it in [5a5a114](https://github.com/iScsc/iscsc.fr/pull/2/commits/5a5a114b5da549c69269408bb913b766b9fe91a0)
Do you think the password policy is too strong?

Reply:
No absolutely not it could even be stronger (10 or 12 charac)

Reply:
Would you want me to increase the policy or can we resolve this?

Reply:
it will be enough for now we can resolve this!